### ISSUE TITLE
no explanation of where things fail?

### ISSUE THREAD

**Issue by Pomax** [tarunc/gulp-jsbeautifier#5]
If I use this module, rather than the grunt version, I get things like:
```
[14:03:22] Using gulpfile ~\Documents\Git proj
[14:03:22] Starting 'jsbeautify'...
[14:03:23] Beautifying lib\analytics.js
[14:03:23] 'jsbeautify' errored after 78 ms
[14:03:23] Error in plugin 'gulp-jsbeautifier'
Beautify failed for: lib\analytics.js
```
without any indication of where/why the failure occured.

**Comment by tarunc** [comment]
Thanks for reporting this. When I created this module, the grunt version didn't report exact errors (where/why) either. The problem is that js-beautify doesn't report exact line numbers where the file is not beautified. Instead, it just gives us if it is pretty or not. However, it makes sense that you'd want to know where and how things failed.

I just released `v0.0.5` of `gulp-jsbeautifier` which should do this. Let me know if this will do for your needs. For example:
![screen shot 2015-03-06 at 11 16 00 pm](https://cloud.githubusercontent.com/assets/602762/6540539/d6fdfac0-c456-11e4-8ee1-e67bb4bfaa68.png)

(The red indicates what is extra in your file. The green indicates what is not currently present in your file but should be for it to be considered pretty.)
